src/preferences/dialog/dlgprefkey.cpp

```diff
@@ -128,8 +128,11 @@ void DlgPrefKey::loadSettings() {
         }
     }
 
-    setNotation(notation_type);
+    // Store notation map for later rećall...
```

Comment:
Why all the accents? Maybe one of your keys got stuck ;)

Reply:
naa, my display now has cracks all over the left side. so usually it's 'why that accent? oh, it's just a crack..'
here it was the other way around : )